submissions/models.py

```diff
@@ -140,6 +140,7 @@ def get_queryset(self):
             return super(Submission.SoftDeletedManager, self).get_queryset().exclude(status=Submission.DELETED)
 
     objects = SoftDeletedManager()
+    _objects = models.Manager()  # Don't use this unless you know and can explain why objects doesn't work for you
```

Comment:
This allows me to get past the SoftDeletedManager filtering, which is necessary:

```
[
    {u'time': u'0.000',
     u'sql': u'SELECT `submissions_submission`.`id`, `submissions_submission`.`uuid`, `submissions_submission`.`student_item_id`, `submissions_submission`.`attempt_number`, `submissions_submission`.`submitted_at`, `submissions_submission`.`created_at`, `submissions_submission`.`raw_answer`, `submissions_submission`.`status` FROM `submissions_submission` ORDER BY `submissions_submission`.`submitted_at` DESC, `submissions_submission`.`id` DESC LIMIT 2'
     },
    {u'time': u'0.000',
     u'sql': u"UPDATE `submissions_submission` SET `uuid` = '524dd0a385a14c749d0a910798efad8e' WHERE `submissions_submission`.`id` = 5"
     },
    {u'time': u'0.000',
     u'sql': u"UPDATE `submissions_submission` SET `uuid` = 'ffe50a97e19345e48f1a62b1401b07e9' WHERE `submissions_submission`.`id` = 4"
     },
    {u'time': u'0.000',
     u'sql': u'SELECT `submissions_submission`.`id`, `submissions_submission`.`uuid`, `submissions_submission`.`student_item_id`, `submissions_submission`.`attempt_number`, `submissions_submission`.`submitted_at`, `submissions_submission`.`created_at`, `submissions_submission`.`raw_answer`, `submissions_submission`.`status` FROM `submissions_submission` ORDER BY `submissions_submission`.`submitted_at` DESC, `submissions_submission`.`id` DESC LIMIT 2 OFFSET 2'
     },
    {u'time': u'0.000',
     u'sql': u"UPDATE `submissions_submission` SET `uuid` = 'd057f32ec283463698b74fe8a5407b13' WHERE `submissions_submission`.`id` = 6"
     },
    {u'time': u'0.000',
     u'sql': u"UPDATE `submissions_submission` SET `uuid` = 'c5f888b8868446bda994c4a7acfeadf9' WHERE `submissions_submission`.`id` = 2"
     },
    {u'time': u'0.000',
     u'sql': u'SELECT `submissions_submission`.`id`, `submissions_submission`.`uuid`, `submissions_submission`.`student_item_id`, `submissions_submission`.`attempt_number`, `submissions_submission`.`submitted_at`, `submissions_submission`.`created_at`, `submissions_submission`.`raw_answer`, `submissions_submission`.`status` FROM `submissions_submission` ORDER BY `submissions_submission`.`submitted_at` DESC, `submissions_submission`.`id` DESC LIMIT 1 OFFSET 4'
     },
    {u'time': u'0.000',
     u'sql': u"UPDATE `submissions_submission` SET `uuid` = '88f6ae6f67f411e7b72e08002779ca96' WHERE `submissions_submission`.`id` = 1"
     },
]
```